src/formats/MMTF.cpp

```diff
@@ -217,6 +232,28 @@ void MMTFFormat::read(Frame& frame) {
                 );
             }
 
+            // Add additional global (not by group) bonds
+            while (bond_index < inter_residue_bond_count) {
+                auto atom1 = static_cast<size_t>(structure_.bondAtomList[bond_index * 2]);
+                auto atom2 = static_cast<size_t>(structure_.bondAtomList[bond_index * 2 + 1]);
+
+                // We are below the atoms we care about
+                if (atom1 < atomSkip_ || atom2 < atomSkip_) {
+                    if (!(atom1 < atomSkip_ && atom2 < atomSkip_)) {
+                        warning("MMTF Reader", "chemfiles can not represent bonds between different models");
+                    }
+                    continue;
+                }
+
+                // We are above the atoms we care about
+                if (atom1 > atomIndex_ || atom2 > atomIndex_) {
+                    break;
```

Comment:
Why did you keep the warning above and removed it here?

Do you know if MMTF allow for bonds between models? (Or if that even make sense?)

Reply:
Technically speaking MMTF could store that, but it would be highly unlikely and probably due to an encoding mistake, not biology. I removed the warning in any case because it doesn't make sense. I think most MMTF encoders don't allow for it anyway.